agent/__init__.py

```diff
@@ -642,6 +648,29 @@ def write_metadata(data, rewrite_file):
     metadata_path.chmod(0o644)
 
 
+def enroll_device(enroll_token, claim_token, device_id):
+
+    payload = {
+        'key': enroll_token,
+        'claim_token': claim_token,
+        'device_id': device_id
+    }
+
+    try:
+        enroll_req = requests.post(
+            '{}/v0.2/enroll-device'.format(WOTT_ENDPOINT),
+            json=payload
+        )
+    except requests.exceptions.RequestException:
+        logger.exception("enroll_device :: rises exception:")
+
+    if not enroll_req.ok:
```

Comment:
`enroll_req` won't be available in case of exception (above)

Reply:
Fixed